app/server/utils/auth.py

```diff
@@ -323,12 +322,12 @@ def create_user_response_object(user, auth_token, message):
         usd_to_satoshi_rate = None
 
     conversion_rate = 1
-    currency_name = current_app.config['CURRENCY_NAME']
+    currency_symbol = user.default_organisation.token.symbol if user.default_organisation and user.default_organisation.token else None
+    currency_decimals = user.default_organisation.token.display_decimals if user.default_organisation and user.default_organisation.token else None
```

Comment:
I don't love this because as a developer coming in, I'd expect that if 
`currency_symbol` is given by `token.symbol`
then 
`currency_decimals` would be given by `token.decimals`, but it's not, which is pretty confusing.

I realise that this naming mess up is somewhat my fault, as it can be traced back to me in Lebanon naming the number of digits displayed behind the decimal place as `display decimals`, but I don't think it makes sense to make the problem any worse.

In an ideal world, I'd suggest that we use currency_decimals everywhere as u've got it here and change and `token.decimals` to `token.precision` or something like that, but we're running up against the ERC20 standard:
https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md#decimals

For now i think the best thing to do is call it display_decimals as much as we can on the backend, and only use  `currency_decimals` at the last minute in the backend.

More long term, (though i think this requires spec'ing), we should probably use  proper i18n module (libraries or hand written) that support  proper formatting. Then we can just have a display format column that stores something like `[SYM] ###,###.##`

Reply:
Hmmm should this PR just be closed in favour of https://github.com/teamsempo/SempoBlockchain/issues/284 @enjeyw ?

Reply:
I still think `display_decimals` is a decent change to implement for now, as #284 is a slightly different thing. Unless u wanted to jump straight into i18n, but that would be a much longer PR